scout/models/case/case_loading_models.py

```diff
@@ -86,18 +94,14 @@ class Sex(str, Enum):
     unknown = "unknown"
 
 
-def _get_demo_file_absolute_path(partial_path: str) -> str:
-    """returns the absolute path to a given demo file."""
-    APP_ROOT: str = path.abspath(path.join(path.dirname(__file__), ".."))
-    return path.join(APP_ROOT, partial_path)
-
-
-def _is_string_path(string_path: str) -> bool:
-    try:
-        path = Path(string_path) or Path(_get_demo_file_absolute_path(partial_path=string_path))
-        return path.is_file()
-    except AttributeError:
-        return False
+def _resource_abs_path(string_path: str) -> str:
+    """Return the absolute path to a resource file."""
+    if exists(string_path) and isabs(string_path):
```

Comment:
you could flatten this conditional as 
```suggestion
    if not exists(string_path):
        raise FileExistsError(f"Path {string_path} could not be found on disk.")
    if isabs(string_path):
        return string_path
    return abspath(string_path)    
```

Reply:
Yes, I like it!

Reply:
Fixed!